Changelog.md

```diff
@@ -3,10 +3,12 @@
 ## 0.0.29
 
 - API based components support
-  - Completion and hover for properties
+  - Completion and hover for properties
```

Comment:
How about changing `hover` to `hover documentation` ? I believe that this will be more clear then to the reader.